tests_integration/nativeapp/test_init_run.py

```diff
@@ -423,18 +422,35 @@ def test_nativeapp_init_from_repo_with_single_template(
 
 # Tests that application post-deploy scripts are executed by creating a post_deploy_log table and having each post-deploy script add a record to it
 @pytest.mark.integration
-def test_nativeapp_app_post_deploy(runner, snowflake_session, project_directory):
+@pytest.mark.parametrize("is_versioned", [True, False])
+def test_nativeapp_app_post_deploy(
+    runner, snowflake_session, project_directory, is_versioned
+):
```

Comment:
When merging the logic, we noticed that post-deploy scripts weren't being called post-upgrade for versioned / release directive install modes. So I parameterized this test to confirm they are now working.